src/github.com/getlantern/launcher/launcher_windows.go

```diff
@@ -26,12 +25,14 @@ func CreateLaunchFile(autoLaunch bool) {
 			log.Errorf("Could not get Lantern directory path: %q", err)
 			return
 		}
-		err = gowin.WriteStringReg("HKCU", runDir, "value", lanternPath)
+		err = gowin.WriteStringReg("HKCU", runDir, "Lantern", "\""+lanternPath+"\""+" -startup")
 		if err != nil {
 			log.Errorf("Error inserting Lantern auto-start registry key: %q", err)
 		}
 	} else {
-		err = gowin.DeleteKey("HKCU", currentVersionDir, "Run")
```

Comment:
This would try to delete the entire Run key, but that includes values from other apps, so this could easily fail.